tests/Httpful/HttpfulTest.php

```diff
@@ -464,7 +464,7 @@ function testMultiHeaders()
         $req = Request::init();
         $response = new Response(self::SAMPLE_JSON_RESPONSE, self::SAMPLE_MULTI_HEADER, $req);
         $parse_headers = $response->_parseHeaders(self::SAMPLE_MULTI_HEADER);
-        $this->assertEquals('Value1,Value2', $parse_headers['X-My-Header']);
+        $this->assertEquals('Value1,Value2', $parse_headers['x-my-header']);
```

Comment:
The culprit is the upstream implementation of Response\Headers::fromString. There the keys are stored lower case. I retained that behavior, and so had to change the test.
Should I drop the lower case?